Makefile

```diff
@@ -42,7 +42,7 @@ FAIL_COVERAGE = { echo '$(COLOUR_RED)FAIL - Coverage below $(COVERAGE)%$(COLOUR_
 .PHONY: check-coverage cover test
 
 # --- Lint ---------------------------------------------------------------------
-GOLINT_VERSION = 1.27.0
+GOLINT_VERSION = 1.31.0
 GOLINT_INSTALLED_VERSION = $(or $(word 4,$(shell golangci-lint --version 2>/dev/null)),0.0.0)
```

Comment:
We can't upgrade past 1.27.0 without fixing the version comparison. After that version, the version string reported by golangci-lint starts with a `v`.

I think it should be enough to wrap this `or` expression wth `$(patsubst v%,%,$(or ...))` - but its "starting" to get ugly.

Reply:
hmm...
I get:
```
golangci-lint --version
golangci-lint has version 1.31.0 built from 3d6d0e7 on 2020-09-07T15:14:41Z
```
no `v` in sight. Am I missing something?

Reply:
No. I remember now. I got this wrong.

When you build yourself from source, you get a `v` prefix. I thought it was a change at some point, but it wasn't. I get the `v` prefix because my locally installed copy is locally-built. Sigh. I will consider how I should fix this myself.